v2/yarpcprotobuf/protoc-gen-yarpc-go/internal/generator/registry.go

```diff
@@ -195,6 +211,40 @@ func (r *registry) loadService(f *File, s *descriptor.ServiceDescriptorProto) er
 	return nil
 }
 
+// loadDependencies collects all of the File's dependencies, both
+// direct and transitive, and assigns them to file.Dependencies.
+func (r *registry) loadDependencies(file *File) error {
+	seen := make(map[string]struct{})
+	files, err := r.loadDependenciesRecurse(file, seen)
+	if err != nil {
+		return err
+	}
+	file.Dependencies = files
+	return nil
+}
+
+func (r *registry) loadDependenciesRecurse(file *File, seen map[string]struct{}) ([]*File, error) {
+	seen[file.descriptor.GetName()] = struct{}{}
+	var deps []*File
+	for _, filename := range file.descriptor.GetDependency() {
+		if _, ok := seen[filename]; ok {
+			continue
+		}
+		f, err := r.getFile(filename)
```

Comment:
naming nit: functions `getFile` and `loadFile` in this go file sound similar but do very different things. Would be good to distinguish the difference here. Maybe even remove `getFile` as its usage is as long as the implementation.

Reply:
I'd prefer to keep the getFile` and `getMessage` functionality since they are used more than once and prevent us from writing the same error message multiple times. My thinking here was that `get` is equivalent to fetching the data, whereas `load` corresponds to populating/mutating the data. 
Does that make sense? I'm happy to update with a better description and/or rename if you still see a need for it.

Reply:
I like it! Updated to `lookupFile` and `lookupMessage`.